### DISCUSSION TITLE
HIgh cluster cmds in jedis client 2.9.0.

### DISCUSSION THREAD

**Discussion by ayushjainswiggy** [xetorthio/jedis#3311] [Q&A]
we are seeing high cluster cmds on one of the shards of our redis cluster. And these cluster command increases with the increase in throughput. We want to know when does jedis client make cluster commands(cluster slot, cluster nodes, etc) and how we can optimize this.

**Comment by sazzad16** [comment]
2.9.0 is pretty old. Please use a recent version.
